accounts/models.py

```diff
@@ -1,101 +1,35 @@
-from django.dispatch import receiver
-from django.contrib.auth.models import User
 from django.db import models
-from django.conf import settings
-from django.db.models.signals import post_delete
-from allauth.account.signals import user_signed_up
+from django.contrib.auth.models import AbstractUser
 
 from .lists import USER_TYPES_CHOICES, LMS_MODULES_CHOICES
-import logging
 
-# Initialise instance of a logger to handle error logging
-logger = logging.getLogger(__name__)
 
-
-class Profile(models.Model):
-    """
-    Define Profile Model with OneToOne relationship to AUTH_USER_MODEL and
-    custom fields to suit Signup Form Extending in forms.py replicating same
-    in DB.
-    Using "related_name" in the OneToOne relationship between the two models
-    specifies the reverse relationship to the User Model, allowing us to
-    target the custom fields for injection into the profile.html
-    template via `{{ user.profile.<<field_name>> }}`.
-    """
-    user = models.OneToOneField(
-        settings.AUTH_USER_MODEL, default=1,
-        related_name='profile',
-        on_delete=models.CASCADE
-    )
+class CustomUser(AbstractUser):
+    """ Custom user model extending the basic AbstractUser model """
     slack_display_name = models.CharField(
         max_length=80,
         blank=False,
-        default=''
+        null=True
```

Comment:
Could you change this to `default=''` again. As mentioned in the Django documentation ([here](https://docs.djangoproject.com/en/3.1/ref/models/fields/#field-options)) this is preferred to setting `null=True`. See